modin/core/storage_formats/pandas/utils.py

```diff
@@ -13,6 +13,7 @@
 
 """Contains utility functions for frame partitioning."""
 
+from modin.config.envvars import MinElementsInPartition
```

Comment:
```suggestion
from modin.config import MinElementsInPartition
```
All objects from `envvars` are available from `modin.config`